libraries/botbuilder-dialogs/tests/numberPrompt.test.js

```diff
@@ -1,4 +1,5 @@
 const { ConversationState, MemoryStorage, TestAdapter } = require('botbuilder-core');
+const { ActivityTypes } = require('botframework-schema');
```

Comment:
This should come through botbuilder-core because the schema is exposed through this library.

If someone forked the repo and then only installed the dependencies for botbuilder-dialogs and then ran the tests, these tests would fail on an import error.